app/_how-tos/kic-get-started-proxy-caching.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Proxy Caching
+description: |
+  Cache all GET and HEAD responses across all Services using Proxy Cache and the KongClusterPlugin resource
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/get-started/proxy-caching/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Get Started
+
+series:
+  id: kic-get-started
+  position: 4
+
+tldr:
+  q: How do I cache upstream responses using {{ site.kic_product_name }}?
+  a: |
+    Use the `proxy-cache` plugin by creating a `KongPlugin` resource while specifying `config.response_code`, `config.request_method` and `config.cache_ttl`.
+
+products:
+  - kic
+
+tools:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+prereqs:
+  skip_product: true
+---
+
+## The Proxy Cache Plugin
+
+One of the ways Kong delivers performance is through caching. The [Proxy Cache plugin](/hub/kong-inc/proxy-cache/) accelerates performance by caching responses based on configurable response codes, content types, and request methods.  When caching is enabled, upstream services are not impacted by repetitive requests, because {{site.base_gateway}} responds on their behalf with cached results. Caching can be enabled on specific routes for all requests globally.
+
+## Proxy Cache Headers
+The `proxy-cache` plugin returns a `X-Cache-Status` header that can contain the following cache results:
+
+{% table %}
+columns:
+  - title: State
+    key: state
+  - title: Description
+    key: desc
+rows:
+  - state: Miss
+    desc: |
+      The request could be satisfied in cache, but an entry for the resource was not found in cache, and the request was proxied upstream.
+  - state: Hit
+    desc: |
+      The request was satisfied and served from the cache.
+  - state: Refresh
+    desc: |
+      The resource was found in cache, but could not satisfy the request, due to Cache-Control behaviors or reaching its hard-coded `cache_ttl` threshold.
+  - state: Bypass
+    desc: |
+      The request could not be satisfied from cache based on plugin configuration
+{% endtable %}
```

Comment:
You should use the include here, and just turn the contents of that include into a table:

```suggestion
{% include_cached /plugins/caching/cache-header.md %}
```